# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-gma500.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

inf. loops on: Sony Corporation VGN-P91S/VAIO, BIOS R2073U3 01/13/2010

[  926.465851]  ? ioread32+0x9/0x27
[  926.465862]  ? REGISTER_READ+0x10/0x12 [gma500_gfx]
[  926.465904]  ? psb_intel_lvds_set_power+0x9e/0xae [gma500_gfx]

--- linux-6.8/drivers/gpu/drm/gma500/psb_intel_lvds.c.vanilla	2024-03-31 12:52:20.044961170 +0200
+++ linux-6.8/drivers/gpu/drm/gma500/psb_intel_lvds.c	2024-03-31 12:48:45.654963758 +0200
@@ -240,14 +240,16 @@
 
 static void psb_intel_lvds_encoder_dpms(struct drm_encoder *encoder, int mode)
 {
+    if (0) {
 	struct drm_device *dev = encoder->dev;
 
 	if (mode == DRM_MODE_DPMS_ON)
 		psb_intel_lvds_set_power(dev, true);
 	else
 		psb_intel_lvds_set_power(dev, false);
 
 	/* XXX: We never power down the LVDS pairs. */
+    }
 }
 
 static void psb_intel_lvds_save(struct drm_connector *connector)
